libraries/AP_HAL_ChibiOS/hwdef/BlitzF745AIO/hwdef.dat

```diff
@@ -3,6 +3,8 @@
 # for IFLIGHT_BLITZ_F7_AIO hardware.
 # thanks to betaflight for pin information
 
+DEFAULTGPIO OUTPUT LOW PULLDOWN
```

Comment:
We do have this in the bootloader. Do we really need this here?

Reply:
We need it for the handoff from bootloader to main firmware